constraintlayout/core/src/main/java/androidx/constraintlayout/core/motion/utils/KeyCycleOscillator.java

```diff
@@ -303,7 +304,7 @@ private static void swap(int[] array, float[] value1, float[] value2, int a, int
         Oscillator mOscillator = new Oscillator();
         private final int OFFST = 0;
         private final int PHASE = 1;
```

Comment:
Why did these not change?

Reply:
Nice catch!  Because *mOffset* and *mPhase* were already declared in line 312 and line 313 respectively. So I left them there for this fix. I will deal with them in my last pass of fix - need to think about good names for both.